srcpkgs/rogue/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'rogue'
+pkgname=rogue
+version=5.4.4
+revision=1
+archs="x86_64-glibc"
```

Comment:
```suggestion
build_style=gnu-configure
configure_args="--enable-scorefile=/var/games/rogue/rogue.scr"
```
you don't need to restrict the archs, it builds just fine on all archs.

also use the build_style

Reply:
Damn, I'd tried using gnu-configure before, but for some reason it didn't build. Also, it's preferred to have save files and the likes of it under /var/games?

Reply:
I'm following the example of a few other packages:

```
$ xlocate var/games
Powermanga-0.93.1_1  /var/games/powermanga/powermanga.hi
Powermanga-0.93.1_1  /var/games/powermanga/powermanga.hi-easy
Powermanga-0.93.1_1  /var/games/powermanga/powermanga.hi-hard
bsd-games-2.17_5     /var/games/bsdgames/hack/data
bsd-games-2.17_5     /var/games/bsdgames/hack/help
bsd-games-2.17_5     /var/games/bsdgames/hack/hh
bsd-games-2.17_5     /var/games/bsdgames/hack/rumors
bsd-games-2.17_5     /var/games/bsdgames/phantasia/characs
bsd-games-2.17_5     /var/games/bsdgames/phantasia/gold
bsd-games-2.17_5     /var/games/bsdgames/phantasia/lastdead
bsd-games-2.17_5     /var/games/bsdgames/phantasia/mess
bsd-games-2.17_5     /var/games/bsdgames/phantasia/monsters
bsd-games-2.17_5     /var/games/bsdgames/phantasia/motd
bsd-games-2.17_5     /var/games/bsdgames/phantasia/scoreboard
bsd-games-2.17_5     /var/games/bsdgames/phantasia/void
lbreakout2-2.6.5_2   /var/games/lbreakout2/lbreakout2.hscr
ltris-1.2.6_1        /var/games/ltris/ltris.hscr
nethack-3.6.7_1      /var/games/nethack/nhdat
nethack-3.6.7_1      /var/games/nethack/symbols
typespeed-0.6.5_2    /var/games/typespeed.score
xlennart-1.1.1_1     /var/games/xlennart.scores.default
```

Reply:
Oh that's a nifty command, thanks!